src/renderer/components/AccountInfo/AccountInfoContainer.tsx

```diff
@@ -178,11 +165,7 @@ const AccountInfoContainer: React.FC<Props> = (props: Props) => {
               ? () => { }
               : onOpenWindow}
           canClaimReward={mcStatus?.rewardInfos != null && mcStatus.rewardInfos.length > 0}
-          goldLabel={
-            mcStatus?.fungibleAssetValue.quantity
-              ? Number(mcStatus.fungibleAssetValue.quantity)
-              : Number(goldAndLevel?.stateQuery.agent?.gold)
-          }
+          goldLabel={Number(collectionStateQuery?.stateQuery.agent?.gold)}
```

Comment:
Does this change the gold? I don't seem to see any code to refetch the Query.

Reply:
[`collectionStateQuery` will be polled.](https://github.com/planetarium/9c-launcher/blob/a103dfa457da25e7f0c6fae1e213c1788a1a4e00/src/renderer/components/AccountInfo/AccountInfoContainer.tsx#L53) so it will be changed.

Reply:
So, now we do not use `subscription` anymore?

Reply:
subscription is still used when we receive messages between polling cycles.